HTML/Taco Tuesday.html

```diff
@@ -0,0 +1,95 @@
+                <tr><td>Saturday</td><td>6 + <strong>A</strong> + <strong>B</strong></td><td>Number of Ports</td></tr>
+                </table></center>
+                <p>Take the value you got from the first field and read the first table from the direction given below. If that food item is not an option in the first field, continue reading in the indicated direction until you reach a food item that is present. That is the food you need to submit in the First Field.</p>
+                <ul><li>If there is a DVI-D Port on the bomb, take the first value that matches when reading from the top down.</li>
+                <li>Otherwise, If there is a lit indicator CAR on the bomb, take the first value that matches when reading from the bottom up.</li>
+                <li>Otherwise, If today is Tuesday, take the first value that matches when reading from the top down.</li>
+                <li>Otherwise, take the first value that matches when reading from the bottom up.</li>
+                </ul>
+                <p>With the Second Field value you calculated, look at the table below to obtain the Day to submit for the Second Field:</p>
+                <center><table><tr><th>Sunday</th><th>Monday</th><th>Tuesday</th><th>Wednesday</th><th>Thursday</th><th>Friday</th><th>Saturday</th></tr>
+                <tr><td>0</td><td>1</td><td>2</td><td>3</td><td>4</td><td>5</td><td>6</td></tr>
+                </table></center>
+                <p>Finally, to obtain the final field check for the following conditions:</p>
+                <ul><li>If there is an Ice Cream module on this bomb, the final field is Sunday.</li>
+                <li>Otherwise, If your first Day value matches the current day or <strong>A</strong> modulo 7, the final field is Tuesday.</li>
+                <li>Otherwise, if the number of ports on the bomb is even, the final field is Thursday.</li>
+                <li>Otherwise, if there are any vowels in the serial number, the final field is Wednesday.</li>
+                <li>Otherwise, if there are no Parallel Ports on the bomb, the final field is Monday.</li>
+                <li>Otherwise, if there is an indicator labelled FRK, the final field is Saturday.</li>
+                </ul>
+            </div>
+            <div class="page-footer relative-footer">Page 2 of 3</div>
+        </div>
+        <div class="page page-bg-03">
+            <div class="page-header">
+                <span class="page-header-doc-title">Keep Talking and Nobody Explodes Mod</span>
+                <span class="page-header-section-title">Taco Tuesday</span>
+            </div>
+            <div class="page-content">
+                <ul><li>Otherwise, the final field is Friday.</li>
+                </ul>
+                <p>However, if both Day Fields are the same day, press the button between them to turn the final field off. We do not like redundant redundancies.</p>
```

Comment:
This spacing / lining up of these lines could be fixed, although it isn't 100% necessary.